packages/google_maps_flutter/google_maps_flutter/test/controller_test.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+import 'fake_google_maps_flutter_platform.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Subscriptions are canceled on dispose',
+      (WidgetTester tester) async {
+    final FakeGoogleMapsFlutterPlatform platform =
+        FakeGoogleMapsFlutterPlatform();
+
+    GoogleMapsFlutterPlatform.instance = platform;
+
+    final FakePlatformViewsController fakePlatformViewsController =
+        FakePlatformViewsController();
+
+    tester.binding.defaultBinaryMessenger.setMockMethodCallHandler(
```

Comment:
What issue was this code added to solve? Tests at this layer should not be relying on the internal implementation details of a (legacy) implementation from another package. Since you're setting the fake instance above it's not clear to me what code could be triggering calls to this handler.